rs/nns/governance/src/governance/tests/mod.rs

```diff
@@ -689,7 +690,8 @@ mod convert_create_service_nervous_system_proposal_to_sns_init_payload_tests_wit
                 ),
                 dapp_canisters: Some(sns_init_pb::DappCanisters {
                     canisters: vec![pb::Canister {
-                        id: Some(CanisterId::from_u64(1000).get()),
+                        // must match `SPECIFIED_CANISTER_ID` defined in `ic-nns-test-utils`
+                        id: Some(CanisterId::from_u64(0x2100000).get())
```

Comment:
If these  have to match, at the very least create another constant that these usages can depend on, and then make a note in both test_utils and wherever this constant should go that they need to currently be kept in sync.

Reply:
Please see: https://github.com/dfinity/ic/pull/5269/commits/2799a7c62727dd132f00b84207ea46d385c30b3a

Reply:
And https://github.com/dfinity/ic/pull/5269/commits/51cdd51b61328b03d84bc3b5f6eaa0c5e87db874 to avoid dead code.